jerry-core/ecma/operations/ecma-arraybuffer-object.c

```diff
@@ -0,0 +1,123 @@
+/* Copyright 2016 Intel Corporation
```

Comment:
Please use the generic copyright notice for the project instead, we are phasing out individual copyright notices:  
"Copyright JS Foundation and other contributors, http://js.foundation" (without the quotes)  
There's more information about that in #1473.